pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -53,11 +53,14 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "inkscape";
-  version = "1.1.2";
+  version = "1.2";
 
-  src = fetchurl {
-    url = "https://media.inkscape.org/dl/resources/file/${pname}-${version}.tar.xz";
-    sha256 = "sha256-P/5UoG0LJaTNi260JFNu8e0gW+E0Q6Oc1DfIx7ibltE=";
+  src = fetchFromGitLab {
+    owner = "inkscape";
+    repo = "inkscape";
+    rev = "dc2aedaf03f19bd890529a9e4720a02465fa1b6c";
```

Comment:
You would need to use `INKSCAPE_${builtins.replaceStrings [ "."] [ "_"] version}`. But it is better to wait for the tarball release as `fetchSubmodules` can sometimes be quite brittle.

Reply:
I attempted fetching the tar.gz source tarball. It seems the submodules aren't included it.

Reply:
Do you mean the official tarball, or the GitLab one? The latter is useless when project contains submodules, just like on GitHub. If the former, then that needs to be reported upstream.

Reply:
The latter. I'll hold out for the official one. Thanks for the heads up.

Reply:
The [official tarball](https://media.inkscape.org/dl/resources/file/inkscape-1.2_2022-05-15_dc2aedaf03.tar.xz) linked from the [release page](https://inkscape.org/release/inkscape-1.2/) comes with submodules included.

Reply:
Updated.